src/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required (VERSION 3.13)
 
 ### Basic compilation settings
 set (CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
I think it's safe to require C++14 as LOVE 12 already did require C++17. Well, can always bump it up if needed.